lib/scripts.js

```diff
@@ -326,17 +326,13 @@ const scripts = {
     return allRemoved;
   },
 
-  extendLock(
-    queue,
-    jobId,
-    duration,
-  ) {
+  extendLock(queue, jobId, duration) {
     return queue.client.extendLock([
       queue.toKey(jobId) + ':lock',
       queue.keys.stalled,
       queue.token,
       duration,
-      jobId,
+      jobId
```

Comment:
This is not following the current format standard, we use trailing commas (as the default in "prettier"): https://stackoverflow.com/questions/61370583/trailing-comma-after-last-line-in-object